lib/trento_web/controllers/cluster_controller.ex

```diff
@@ -55,4 +55,34 @@ defmodule TrentoWeb.ClusterController do
         |> json(%{error: reason})
     end
   end
+
+  @spec get_connection_settings(Plug.Conn.t(), map) :: Plug.Conn.t()
+  def get_connection_settings(conn, %{"cluster_id" => cluster_id}) do
+    settings = Clusters.get_hosts_connection_settings(cluster_id)
```

Comment:
Should we have some `non happy path` scenario handling here?

Reply:
the unhappy path here is something really bad with the database, which will raise.
There's no `{:error, something}` path here.